### DISCUSSION TITLE
External Position Dependent Force Fields

### DISCUSSION THREAD

**Discussion by anisha-yeddanapudi** [glotzerlab/hoomd-blue#1943] [Support] [answered]
Hi,
I was wondering if Hoomd offers the ability to set custom particle position-dependent electric/magnetic fields as part of the molecular dynamics simulations. The closest thing I have found has been `md.external.field`, however, those implementations seem independent of particle position. Thanks!

**Comment by joaander** [comment, marked as answer]
This howto guide describes your options: https://hoomd-blue.readthedocs.io/en/v4.9.1/howto/custom-md-potential.html
